snuba/utils/streams/kafka.py

```diff
@@ -152,6 +253,8 @@ def close(self, timeout: Optional[float] = None) -> None:
         except RuntimeError:
             pass
```

Comment:
Why catching these ?

Reply:
This got changed in #517, specifically as part of 54388f5d12e1e380282e9a2768ebd79833ac2d57 since `close` now could be called multiple times if a `TimeoutError` was raised on flush. `close` on a closed consumer is really just a no-op so it seemed safe to call multiple times as long as it is always the terminal state.